cfgov/unprocessed/js/routes/on-demand/simple-chart/chart-hooks.js

```diff
@@ -16,6 +22,16 @@ const hooks = {
     } ) );
   },
 
+  ccpi_quarterLabels() {
+    const { x, y, series } = this;
+    const d = new Date( x ).toLocaleString(
+      'en-US', { dateStyle: 'medium', timeZone: 'UTC' }
+    ).split( ', ' );
+    const quarter = ccpi_quarterMap[d[0]];
+    const year = d[1];
+    return `<b>${ series.name }</b><br/><b>${ quarter } ${ year }</b><br/>Percentile: <b>${ y }</b>`;
```

Comment:
I must be testing this wrong, since I'm seeing `Cannot read property 'name' of undefined`. When you get back, I'll ask for your help knowing how to test this.

Reply:
Sorry, should have included testing directions!

Add the following to Style overrides:

```
{
"tooltip.formatter": "fn__ccpi_quarterLabels"
}
```

Reply:
Cool, that looks good 👍 . Now, how do I test the series labeling?

Reply:
Set the following data series `[{"key": "index_10percentile", "label":"10th percentile"},"index_25percentile"]`

Reply:
(or make the label whatever you like). You can see that it supports a mix of key/val objects or just plain strings